client/src/components/Workflow/Editor/Node.vue

```diff
@@ -175,7 +175,9 @@ const workflowOutputs = computed(() => props.step.workflow_outputs || []);
 const connectionStore = useConnectionStore();
 const stateStore = useWorkflowStateStore();
 const stepStore = useWorkflowStepStore();
-const isLoading = computed(() => Boolean(stateStore.getStepLoadingState(props.id)?.loading));
+const isLoading = computed(() =>
+    Boolean(stateStore.getStepLoadingState(props.id)?.loading && props.step.outputs.length === 0)
```

Comment:
I'd like to push back that it is important for users to see this. Similarly on every keypress when entering a parameter. 

![wf-clicks](https://user-images.githubusercontent.com/1814954/215154398-809229dc-b81a-4cea-b191-59aa748572f6.gif)

It should also be questioned that the step changes into "loading" state when these things happen, since there is no update. So the only thing we are notifying the user about is an async request to server...I don't think we should generally do that. Another [comment](https://github.com/galaxyproject/galaxy/issues/15323#issuecomment-1405373480) on it.

Reply:
> I'd like to push back that it is important for users to see this. Similarly on every keypress when entering a parameter.

The server may respond with whatever, making current work invalid. I talked with @guerler and he'll have a good fix for the case that nothing has changed on activating the form.

Reply:
imho users are used to things being asynchronous. At time errors are detected, they are displayed within proper context, I don't think bridging the gap with loading indicator is helpful here. I would maybe get if the node and its controls turn `disabled` until the request is done to prevent conflicts, but even at that point it feels like we wouldn't be using the async benefits to its fullest.

Reply:
If we have a disabled indicator for the nodes that would be fine with me, as long as it's clear that the server hasn't responded yet. Cause again, you may be making a connection that will be undone once the response comes in.